packages/nuekit/src/init.js

```diff
@@ -1,22 +1,21 @@
 
 import { compileFile as nueCompile} from 'nuejs-core'
-import { join, basename } from 'node:path'
+import { dirname, join } from 'node:path'
+import { fileURLToPath } from 'node:url'
 import { promises as fs } from 'node:fs'
 import { resolve } from 'import-meta-resolve'
 import { buildJS } from './builder.js'
-import { colors } from './util.js'
+import { colors, srcdir } from './util.js'
 
 
 export async function init({ dist, is_dev, esbuild, force }) {
 
   // directories
   const cwd = process.cwd()
-  const srcdir = getSourceDir()
   const outdir = join(cwd, dist, '@nue')
 
-
   // has all latest?
-  const latest = join(outdir, '.043')
+  const latest = join(outdir, '.05')
   try {
     if (force) doError()
```

Comment:
```suggestion
    if (force) throw new Error()
```

I find this is more readable. `doError()` could be a function. this not.
but it's your decision.

Reply:
Agreed. `throw new Error()` reads better